doc/user/content/ingest-data/sql-server/self-hosted.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Ingest data from self-hosted SQL Server"
+description: "How to stream data from self-hosted SQL Server database to Materialize"
+menu:
+  main:
+    parent: "sql-server"
+    name: "Self-hosted"
+    identifier: "sql-server-self-hosted"
+---
```

Comment:
If we remove the kafka+debezium page, might as well add a redirect.
 
```suggestion
aliases:
  - /ingest-data/cdc-sql-server/
 --- 
```

We do get decent traffic to the cdc-sql-server page ... but, removing it and redirecting to here could get people to try the better path sooner.

Alternatively, we could leave the cdc-sql-server page but just don't show it in the left-hand nav.

Reply:
Good idea! Added!